lib/feature/login_with_email/login_with_email_screen.dart

```diff
@@ -1,22 +1,107 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:task_manager/base/bloc/bloc_provider.dart';
+import 'package:task_manager/constants/edge_insets.dart';
+import 'package:task_manager/constants/size_box.dart';
+import 'package:task_manager/shared/widgets/icons/persion_add_icon.dart';
+import 'package:task_manager/shared/widgets/icons/trello_icon.dart';
 
 class LoginWithEmailScreen extends ConsumerWidget {
   const LoginWithEmailScreen({super.key});
-
   @override
   Widget build(BuildContext context, ref) {
     final bloc = ref.watch(BlocProvider.loginWithEmail);
-    return SafeArea(
-      child: Scaffold(
-        body: InkWell(
-          onTap: () => bloc.loginEmail(
-            email: 'hlndzzkum@gmail.com',
-            password: '123456',
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          SizedBoxConstants.h36,
+          Container(
+            width: MediaQuery.of(context).size.width,
+            height: 100,
+            padding: EdgeInsetsConstants.top50,
+            child: const Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                TrelloIcon(width: 50),
+                SizedBoxConstants.w16,
+                Text(
+                  'Task Manager',
+                  style: TextStyle(
+                    color: Colors.black,
+                    fontSize: 25,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ],
+            ),
+          ),
+          SizedBoxConstants.h36,
+          const Text(
+            'Chọn hoặc thêm tài khoản khác',
+            style: TextStyle(fontWeight: FontWeight.bold),
+          ),
+          SizedBoxConstants.h8,
+          SizedBox(
+            width: MediaQuery.of(context).size.width - 200,
```

Comment:
cái này chỉ xài theo % không nên add cứng 
VD: width: MediaQuery.of(context).size.width * 0.2